### PR TITLE
Lpa 3259 progress bar introduced in status description pages

### PR DESCRIPTION
**Pull request by SeemaMenon**

### CODE REVIEW

File: assets/sass/patterns/_progress-bar.scss [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+  color: $white;
+}
+
+.current-checking ::before {
+  background: $status-checking-colour;
+  border: 2px solid $white;
+  color: $white;
+}
+
+.current-returned ::before {
```
[the remark this answers is not in the document]
Comment (taylorsj1980) [reply assets/sass/patterns/_progress-bar.scss]: This is not a fault.

File: assets/sass/patterns/_progress-bar.scss [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+  color: $white;
+}
+
+.current-received ::before {
+  background: $status-received-colour;
+  border: 2px solid $white;
+  color: $white;
+}
+
+.current-checking ::before {
```
[the remark this answers is not in the document]
Comment (taylorsj1980) [reply assets/sass/patterns/_progress-bar.scss]: This is not a fault.

File: assets/sass/patterns/_progress-bar.scss [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+  }
+}
+
+.current-waiting ::before {
+  background: $status-waiting-colour;
+  border: 2px solid $white;
+  color: $white;
+}
+
+.current-received ::before {
```
[the remark this answers is not in the document]
Comment (taylorsj1980) [reply assets/sass/patterns/_progress-bar.scss]: This is not a fault.

File: assets/sass/patterns/_progress-bar.scss [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+      display: inline-block;
+      height: 30px;
+      line-height: 30px; // make the line-height the same as the height so that number aligns vertically
+      text-align: center;
+      width: 30px;
+    }
+  }
+}
+
+.current-waiting ::before {
```
[the remark this answers is not in the document]
Comment (taylorsj1980) [reply assets/sass/patterns/_progress-bar.scss]: This is not a fault.

File: assets/sass/settings/_variables.scss [hunk 5]
```diff
@@ -5,9 +5,13 @@ $path: '/assets/v2/images/icons/';
 // then replicated for consistency
 $turquoise-with-contrast: #218278;
 $white: #ffffff;
+$black: #0b0c0c;
```
[the remark this answers is not in the document]
Comment (SeemaMenon) [reply assets/sass/settings/_variables.scss]: agree

**Pull request merged**